apps/condo/domains/acquiring/components/payments/PaymentsTable.tsx

```diff
@@ -322,7 +322,7 @@ const PaymentsTable: React.FC<IPaymentsTableProps> = ({ billingContext, contexts
                 </Typography.Text>
             </Modal>
 
-            <MultipleFiltersModal/>
+            {MultipleFiltersModal}
```

Comment:
Why passing component like this?

Reply:
if call component, then component rerender and reset selected template. 
If use just jsx, then this doesn't happen

Reply:
I tried passing the `key` for MultipleFiltersModal like `<MultipleFiltersModal key='modal' />`, but that didn't help either.